modules/geolocation/geolocationWorker.py

```diff
@@ -61,7 +61,7 @@ def geolocation_locator_worker(pause, exitRequest, pipelineIn, pipelineOut, pipe
     return
 
 
-def geolocation_output_worker(pause, exitRequest, pipelineIn, pipelineOut, pipelineInLock):
+def geolocation_output_worker(pause, exitRequest, pipelineIn, pipelineInLock):
```

Comment:
Add a comment why this function signature breaks convention (basically how it outputs to file instead of pipeline queue).